rest/config_test.go

```diff
@@ -117,8 +118,8 @@ func TestConfigValidation(t *testing.T) {
 			assert.NoError(tt, err)
 			errorMessages := config.setupAndValidateDatabases()
 			if test.err != "" {
-				require.Len(t, errorMessages, 1)
-				assert.EqualError(tt, errorMessages[0], test.err)
+				require.NotNil(t, errorMessages)
+				assert.EqualError(tt, errorMessages.Errors[0], test.err)
```

Comment:
We still need to make sure we have at least 1 error in here to avoid a panic.
Should be able to use the Len method on multierror itself?

https://godoc.org/github.com/hashicorp/go-multierror#Error.Len

Reply:
Sure, this makes sense.